Demo/Measures-cms/CumulativeMedicationDuration-4.0.000.cs

```diff
@@ -140,6 +140,30 @@ public CumulativeMedicationDuration_4_0_000(CqlContext context)
 
     #endregion
 
+	private CqlCode[] V3TimingEvent_Value()
+	{
+		var a_ = new CqlCode[0]
```

Comment:
This looks wrong, in the old version there were codes returned here, but now it is empty. (etcetera)